### PR TITLE
Sub static fox image when WebGL isn't supported 

### LINKED ISSUE #548: Create substitute for webgl fox
On Linux Chrome, WebGL is disabled for some reason. Thus, having a static fox substitute would be a small UI improvement. Currently, at lock page, I don't see a fox at all and it looks kinda sad :(

### PR DESCRIPTION
**Pull request by 2-am-zzz**
Fixes #548 by allowing users to pass in a static image to the metamask-logo module.

### CODE REVIEW

**Comment by Zanibas** [comment]
Ready to merge!

File: package.json [hunk 1]
```diff
@@ -45,13 +45,14 @@
     "eth-store": "^1.1.0",
     "ethereumjs-tx": "^1.0.0",
     "ethereumjs-util": "^4.4.0",
+    "gulp": "github:gulpjs/gulp#4.0",
```
Comment (flyswatter) [inline package.json]: We no longer require this weird version of gulp, as long as you're using node 6+, you should be able to use the main version on npm.

File: package.json [hunk 2]
```diff
@@ -76,7 +77,7 @@
     "three.js": "^0.73.2",
     "through2": "^2.0.1",
     "vreme": "^3.0.2",
-    "web3": "ethereum/web3.js#0.16.0",
+    "web3": "github:ethereum/web3.js#0.16.0",
```
Comment (flyswatter) [inline package.json]: Why are we using a weird version of web3? Why the github prefix now?

**Comment by flyswatter** [comment]
Just waiting for the removal of that gulp item from the package.json to merge.

**Comment by Zanibas** [comment]
Done!

**Pull request merged**